Library/ViewModels/PledgePaymentMethodsViewModel.swift

```diff
@@ -354,6 +354,9 @@ public final class PledgePaymentMethodsViewModel: PledgePaymentMethodsViewModelT
 
           if featureStripeLinkEnabled() {
             configuration.defaultBillingDetails.email = AppEnvironment.current.currentUserEmail
+            configuration.billingDetailsCollectionConfiguration.email = .automatic
+          } else {
+            configuration.billingDetailsCollectionConfiguration.email = .always
```

Comment:
@ycheng-kickstarter Can you confirm this is what Android is doing? Stripe's documentation is stunningly unclear on this, but it does work for me.

Reply:
Yup we match